bundles/org.openhab.binding.wiz/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,204 @@
+			<channel id="last-update" typeId="last-update"/>
+		</channels>
+		<config-description-ref uri="thing-type:wiz:device"/>
+	</thing-type>
+
+	<thing-type id="fan">
+		<label>WiZ Ceiling Fan</label>
+		<description>Supports WiZ Ceiling Fans</description>
+
+		<channels>
+			<channel id="state" typeId="system.power"/>
+			<channel id="speed" typeId="fan-speed"/>
+			<channel id="reverse" typeId="fan-reverse"/>
+			<channel id="mode" typeId="fan-mode"/>
+		</channels>
+		<config-description-ref uri="thing-type:wiz:device"/>
+	</thing-type>
+
+	<thing-type id="fan-with-dimmable-bulb">
+		<label>WiZ Ceiling Fan With Dimmable Bulb</label>
+		<description>Supports WiZ Ceiling Fans With a Dimmable Bulb</description>
+
+		<channel-groups>
+			<channel-group id="device" typeId="device-channels"/>
+			<channel-group id="light" typeId="dimmable-light"/>
+			<channel-group id="fan" typeId="fan-group"/>
+		</channel-groups>
+		<config-description-ref uri="thing-type:wiz:device"/>
+	</thing-type>
+
+	<channel-type id="light-mode">
+		<item-type>String</item-type>
+		<label>Light Mode</label>
+		<tags>
+			<tag>Lighting</tag>
+		</tags>
+		<state pattern="String" readOnly="false">
+			<options>
+				<option value="1">Ocean</option>
+				<option value="2">Romance</option>
+				<option value="3">Sunset</option>
+				<option value="4">Party</option>
+				<option value="5">Fireplace</option>
+				<option value="6">Cozy White</option>
+				<option value="7">Forest</option>
+				<option value="8">Pastel Colors</option>
+				<option value="9">Wakeup</option>
+				<option value="10">Bed Time</option>
+				<option value="11">Warm White</option>
+				<option value="12">Daylight</option>
+				<option value="13">Cool White</option>
+				<option value="14">Night Light</option>
+				<option value="15">Focus</option>
+				<option value="16">Relax</option>
+				<option value="17">True Colors</option>
+				<option value="18">TV Time</option>
+				<option value="19">Plant Growth</option>
+				<option value="20">Spring</option>
+				<option value="21">Summer</option>
+				<option value="22">Fall</option>
+				<option value="23">Deep Dive</option>
+				<option value="24">Jungle</option>
+				<option value="25">Mojito</option>
+				<option value="26">Club</option>
+				<option value="27">Christmas</option>
+				<option value="28">Halloween</option>
+				<option value="29">Candlelight</option>
+				<option value="30">Golden White</option>
+				<option value="31">Pulse</option>
+				<option value="32">Steampunk</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="light-mode-speed">
+		<item-type>Dimmer</item-type>
+		<label>Dynamic Light Mode Speed</label>
+		<description>Speed of color/intensity changes in dynamic light modes</description>
+		<tags>
+			<tag>Lighting</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="fan-speed">
+		<item-type>Number</item-type>
+		<label>Fan Speed</label>
```

Comment:
Is there a list of recommended categories somewhere? Or should it just correspond to a semantic tag? And if so, how does that map - Fan is an equipment (group), whereas a channel would be for a single point/item.

Reply:
> Is there a list of recommended categories somewhere?

I don't think so.

> Or should it just correspond to a semantic tag?

I don't think it has any relation to semantic tags, since this concept predates that.

It seems quite tightly coupled with icons in the UI, so usually I just have have a look at https://www.openhab.org/docs/configuration/iconsets/classic/ and check if anything makes sense. For example, `DateTimeType` channels are usually `Time`, batter-level channels are usually `Battery` and temperature channels are usually `Temperature`.

You don't have to provide a category, I just remembered that `Fan` is available, and thought it might be worth considering. But I'm not an expert in general rules/recommendations for mapping these old categories to channel types.

Reply:
I had thought is was classic icons, except the casing doesn't match. I'm happy to provide the category.